modules/test/playwright/pages/users-admin-web/AccountSettingsPage.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class AccountSettingsPage {
+    readonly accountSettingsMenuItem: Locator;
+    readonly page: Page;
+    readonly personalMenuButton: Locator;
+    readonly rolesMenuItem: Locator;
+    readonly saveButton: Locator;
+
+    constructor(page: Page) {
+        this.accountSettingsMenuItem = page.getByRole('menuitem', {
+                name: 'Account Settings'
+            });
+        this.personalMenuButton = page.getByLabel('Test Test User Profile');
```

Comment:
This is not exact enough. This test will fail if we ever decide to change the name from "Test Test". Please use .getByTestId() which is best, or something more specific than getByLabel(). Feel free to message me if you need assistance with this part.